### PR TITLE
chore: upgrade vscode to 1.68.1

### PR DESCRIPTION
**Pull request by conwnet**

### CODE REVIEW

**Comment by Siddhant-K-code** [comment]
Following test is failing: 
- https://github.com/conwnet/github1s/runs/6955360093?check_suite_focus=true#step:10:75 (`TimeoutErr`)
- Conflicting Node version? : https://github.com/conwnet/github1s/runs/6955360092?check_suite_focus=true#step:7:28

**Comment by conwnet** [comment]
> Following test is failing:
> 
> * https://github.com/conwnet/github1s/runs/6955360093?check_suite_focus=true#step:10:75 (`TimeoutErr`)
> * Conflicting Node version? : https://github.com/conwnet/github1s/runs/6955360092?check_suite_focus=true#step:7:28

Thanks you for your reminder, now i fixed it

File: vscode-web/src/vs/editor/common/config/editorOptions.ts [hunk 1]
```diff
@@ -3659,7 +3682,7 @@ export interface IGuidesOptions {
 	 * Enable highlighting of the active indent guide.
 	 * Defaults to true.
 	 */
-	highlightActiveIndentation?: boolean;
+	highlightActiveIndentation?: boolean | 'always';
```
Comment (Siddhant-K-code) [inline vscode-web/src/vs/editor/common/config/editorOptions.ts:3685]: explicit defn. is required? or we can drop this?

File: vscode-web/src/vs/editor/common/config/editorOptions.ts [hunk 2]
```diff
@@ -3736,7 +3766,7 @@ class GuideOptions extends BaseEditorOption<EditorOption.guides, IGuidesOptions,
 			highlightActiveBracketPair: boolean(input.highlightActiveBracketPair, this.defaultValue.highlightActiveBracketPair),
 
 			indentation: boolean(input.indentation, this.defaultValue.indentation),
-			highlightActiveIndentation: boolean(input.highlightActiveIndentation, this.defaultValue.highlightActiveIndentation),
+			highlightActiveIndentation: primitiveSet(input.highlightActiveIndentation, this.defaultValue.highlightActiveIndentation, [true, false, 'always']),
```
Comment (Siddhant-K-code) [inline vscode-web/src/vs/editor/common/config/editorOptions.ts:3769]:
```suggestion
			highlightActiveIndentation: primitiveSet(input.highlightActiveIndentation, this.defaultValue.highlightActiveIndentation, [true, 'always', false]),
```

nit: consistent defn.

File: .github/workflows/test-wtih-vscode-build.yml [hunk 3]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        node-version: [14.x, 16.x]
+        node-version: [16.x]
```
Comment (Siddhant-K-code) [inline .github/workflows/test-wtih-vscode-build.yml:16]:
```suggestion
        node-version: [16.x, 18.x]
```

Add `node18` to VsCode Build Test

File: vscode-web/src/vs/editor/common/config/editorOptions.ts (hunk 1 above)
Comment (conwnet) [reply vscode-web/src/vs/editor/common/config/editorOptions.ts:3685]: Thank you for you suggestion, the code here is sync from vscode, I think we can make it we can make it consistent with the upstream for future updates.

File: vscode-web/src/vs/editor/common/config/editorOptions.ts (hunk 2 above)
Comment (conwnet) [reply vscode-web/src/vs/editor/common/config/editorOptions.ts:3769]: Here too. The code modified by us will be surround with this comment: `/* below codes are changed by github1s */`

File: .github/workflows/test-wtih-vscode-build.yml (hunk 3 above)
Comment (conwnet) [reply .github/workflows/test-wtih-vscode-build.yml]: I've already tried here. And VS Code 1.68.1 recommend that we should use the node.js with version `>=16.14.x and <17`. If we add node 18 here, buiding process will throw an warning.

**Comment by Siddhant-K-code** [comment]
Build for Node 18 is failing because of an outdated version of `@azure/msal-node`. -- Updating it

File: .github/workflows/test-wtih-vscode-build.yml [hunk 4]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-20.04]
-        node-version: [14.x, 16.x]
+        node-version: [16.x, 18.x]
```
Comment (Siddhant-K-code) [inline .github/workflows/test-wtih-vscode-build.yml:16]:
```suggestion
        node-version: [16.x]
```

**Review by Siddhant-K-code — APPROVED** [review]
LGTM ✨

**Review by xcv58 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**